src/Azure/Azure.Quantum.Client/generated/IJobsOperations.cs

```diff
@@ -8,7 +8,7 @@
 // regenerated.
 // </auto-generated>
 
-namespace Microsoft.Azure.Quantum.Client
+namespace Microsoft.Quantum
```

Comment:
Was it intentional to change the namespace of the generated code? My feeling is that "Azure" should still be in there somewhere.

Reply:
i used the swagger we currently have. I believe @anpaz-msft updated it to Microsoft.Quantum

Reply:
The namespace can be specified via the `--namespace` argument to `autorest`. I think the reason for previously using `--namespace=Microsoft.Azure.Quantum.Client` was to be consistent with the package name, which is `Microsoft.Azure.Quantum.Client`.

Reply:
i synced with Andres and reverted it.